### PR TITLE
Fix make lint-helm

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -421,7 +421,7 @@ lint-yaml: ## runs linters against YAML files
 .PHONY: lint-helm
 lint-helm: ## runs linters against helm charts
 	@echo "Helm version: `helm version`"
-	@${FINDFILES} -name 'Chart.yaml' -path './resources/helm/v3.?/*' \
+	@${FINDFILES} -name 'Chart.yaml' -path './resources/charts/v3.?/*' \
```
Comment (jwendell) [inline Makefile:424]: do we need the v3 part?

Comment (luksa) [reply Makefile:424]: only time will tell :)

Comment (jwendell) [reply Makefile:424]: Thinking ahead, when we will support multiple versions, doesn't make sense to cover all versions?

**Review by jwendell — APPROVED** [review]

**Pull request merged**